coords/resolver.py

```diff
@@ -104,16 +106,16 @@ def query_url(name, url, method):
     # different.
     #
     try:
-        v5(f"{method} name query: {url}")
-        return urlopen(url)
+        v5(f"{method} name query: {url}    context is None: {context is None}")
```

Comment:
I assume that `{context is None`} works w/ f strigs, but it does read funny.   As long as it works, should be ok though.

Reply:
Yeah. It works

```
>>> context = None
>>> f"Is context None: {context is None}"
'Is context None: True'
>>> context = True
>>> f"Is context None: {context is None}"
'Is context None: False'
```